stable/yugaware/values.yaml

```diff
@@ -30,8 +30,8 @@ image:
 
   nginx:
     registry: ""
-    tag: 1.17.4
-    name: nginx
+    tag: 1.17.4-amd64
+    name: nginxinc/nginx-unprivileged
```

Comment:
minor: can you comment if the images work the exact same way (entrypoint, config) other than the fact that it is using non-root-owned dirs internally? so if an existing non-openshift user is using a copy of the nginx image from their local enterprise repo, they could continue to use it by setting it to the previous values and nothing would break.

Reply:
Yes, that is correct. They can continue to use the old image. I cross checked this by setting the value.